gazebo_ros_control/src/default_robot_hw_sim.cpp

```diff
@@ -181,6 +181,10 @@ bool DefaultRobotHWSim::initSim(
       return false;
     }
 
+    if(hardware_interface == "EffortJointInterface" || hardware_interface == "PositionJointInterface" || hardware_interface == "VelocityJointInterface") {
+      ROS_WARN_STREAM("Please update old, now deprecated, hardware_interface syntax in joint '" << joint_names_[j] << "': " << hardware_interface);
```

Comment:
How about help the user understand better how to fix the warning? Using something like:

"Deprecated syntex, please prepend 'hardware_interface::' to the JOINT_NAME hardware interface HARDWARE_INTERFACE."